capstone/src/main/webapp/group.html

```diff
@@ -3,17 +3,41 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link
       href="http://fonts.googleapis.com/css?family=Roboto"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="styles/group.css"></link>
-    <script src="scripts/poll.js"></script>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="scripts/post.js"></script>
+    <script src="scripts/poll.js"></script>
     <title>Group</title>
   </head>
 
+<<<<<<< HEAD
```

Comment:
nit: Make sure you remove this after the merge!

Reply:
Oh my bad, I fixed the merge errors on my other branch but not on this one. I merged our HTML code, will fix formatting of UI later.